apps/api/src/schema/programs.ts

```diff
@@ -2,6 +2,27 @@ import { z } from "@hono/zod-openapi";
 
 const programIdBase = z.string({ required_error: "programId is required" });
 
+export const majorQuerySchema = z.object({
+  id: z.string().optional().openapi({
+    description: "The ID of a single major to request, if provided",
+    example: "BA-163",
+  }),
+});
+
+export const minorQuerySchema = z.object({
+  id: z.string().optional().openapi({
+    description: "The ID of the specific minor being requested for",
```

Comment:
This should be reworded consistent with the description for the equivalent field on the majors query.